vela-templates/definitions/internal/workflowstep/collect-service-endpoints.cue

```diff
@@ -0,0 +1,63 @@
+import (
+	"vela/op"
+	"vela/ql"
+)
+
+"collect-service-endpoints": {
+	type: "workflow-step"
+	annotations: {}
+	labels: {
+		"ui-hidden": "true"
```

Comment:
Remove this label, the new step should be used in UI.

Reply:
OK